include/aos/common/serialnum.hpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_SERIALNUM_HPP_
+#define AOS_SERIALNUM_HPP_
+
+#include "aos/common/config.hpp"
+#include "aos/common/tools/string.hpp"
+
+namespace aos {
+namespace serialnum {
+
+/**
+ *  Serial number size(in bytes).
+ */
+constexpr auto cSerialNumLen = AOS_CONFIG_SERIAL_NUM_SIZE;
```

Comment:
Don't think we need this common class for serial numbers. It looks like it will be just hex to string and string to hex converter. So, it should be part of string converters.